easybuild/easyconfigs/h/HapCUT2/HapCUT2-v1.3.3-GCC-12.3.0.eb

```diff
@@ -0,0 +1,75 @@
+easyblock = 'MakeCp'
+name = 'HapCUT2'
+version = 'v1.3.3'
```

Comment:
this version is not installed, but a specific commit, so you should use the date the commit was merged as the version.

Reply:
I will try to patch instead to build from commit as @Flamefire  suggest it.